lambdas/db-migration/main.tf

```diff
@@ -76,6 +76,7 @@ resource "aws_lambda_function" "db_migration" {
   environment {
     variables = {
       databaseCredentialSecretArn = var.rds_user_access_secret_arn
+      dbHeartBeat                 = "true"
```

Comment:
Setting this to true by default given it's not a scaling lambda, disabling it won't really give us much in terms of performance/overhead savings